app/interactors/add_movie_to_round.rb

```diff
@@ -3,26 +3,42 @@
 class AddMovieToRound
   include Hanami::Interactor
 
-  def initialize(import_service: ImportTmdbMovie.new)
-    @import_service = import_service
+  expose :movie, :round
```

Comment:
We must declare the meaningful result fields, stored, in the interactor object, as instance variables.